executor/bin/prepare-executor.sh

```diff
@@ -29,7 +29,7 @@ if [ -z "${TARGET_DIR}" ]; then
 fi
 
 COOK_EXECUTOR_NAME="cook-executor-${MODE}"
-if [[ ! "${MODE}" =~ ^(docker)$ ]]; then
+if [[ "${MODE}" =~ ^(docker)$ ]]; then
```

Comment:
Why are you using a regular expression here? Using the basic test command seems simpler and clearer:
```bash
if [ "${MODE}" == docker ]; then
```